### ISSUE TITLE
discrepancy in documentation.

### ISSUE THREAD

**Issue by carloalbrile** [trustcrypto/OnlyKey-App#219]
Hello,

Version:
App v5.5.0
OnlyKey v3.0.4-prodc
Running on Linux Mint 21.2

Presenting issue:
While attempting to:
Setup/Change Primary Pin
We are instructed to:
"To do this hold down button #6 on your OnlyKey for 5+ seconds and release. The light will turn off. You will notice the OnlyKey flashes red in config mode."
This is the message I see but when I copy and past to document the issue I see:

"To do this hold down button #6 on your OnlyKey for 5+ seconds and release. hold down button #1 on your OnlyKey DUO for 10+ seconds and release.The light will turn off. If a PIN was previously set, re-enter the PIN to enter config mode.You will notice the OnlyKey flashes red in config mode."

Please kindly help to get the correct instructions presented.

Screen shot:
![Screenshot at 2024-01-16 22-45-43](https://github.com/trustcrypto/OnlyKey-App/assets/70098762/48fdc8d8-89dd-4e30-a75c-5863da228bf6)

**Comment by ror3d** [comment]
Indeed, the message in the app should specify, after "The light will turn off", that the user should input their PIN. Otherwise, it doesn't enter config mode and the light doesn't flash red.

**Comment by carloalbrile** [comment]
It took me some time to discover it.
Strange that the text is not displayed but if high light it, copy and past it in a note pad the text is there. So people that wrote the instruction want that inf to be available but for some reason it is not displayed.
Do you know how this can be corrected?
Do you know if this is the correct place to escalate this issue?
thanks.
